docs/project/dependency/oracle-aws.mdx

```diff
@@ -0,0 +1,193 @@
+import Zoom from "react-medium-image-zoom";
+import "react-medium-image-zoom/dist/styles.css";
+import { FiExternalLink } from "react-icons/fi";
+
+# AWS Oracle
+
+SleakOps facilitates the integration of Oracle databases through Amazon RDS (Relational Database Service). Amazon RDS for Oracle is a fully managed service that simplifies database setup, operation, and scaling. By leveraging SleakOps for this integration, you can efficiently manage Oracle databases within your EKS (Elastic Kubernetes Service) environment, ensuring robust, scalable, and reliable data storage solutions for your applications.
+
+
+## FAQs 
+<details>
+<summary>
+### License
+</summary>
+When creating an Oracle DB using Sleakops License Included (LI). Currently Bring Your Own License (BYOL) is not supported, however, you can manually change it in the AWS Console.
+</details>
+
+
+<details>
+<summary>
+### How does SleakOps manage Oracle credentials?
+</summary>
+When you create an Oracle dependency in SleakOps, it automatically generates a Vargroup for your database. This Variable Group securely stores the Oracle credentials and other important configuration details, such as the database endpoint and user access information. You'll be able of manage them from  [Vargroups](/project/vargroup/index.mdx) section.
+</details>
+
+<details>
+<summary>
+### What is Multi-AZ deployment and should I enable it?
+</summary>
+Multi-AZ (Availability Zone) deployment ensures high availability and failover support by replicating your database in another availability zone. It's recommended for production environments to prevent downtime. Keep in mind that it increases costs.
+</details>
+
+<details>
+<summary>
+### Can I change the Oracle version after the database is deployed?
+</summary>
+No, the database engine version cannot be changed after deployment. You would need to create a new Oracle instance with the desired version and migrate your data. Or change it manually into the AWS Console.
+</details>
+
+<details>
+<summary>
+### What happens if I need more storage for my Oracle database?
+</summary>
+You can adjust the storage size when configuring your database. If you need more storage after deployment, you can scale modifying the settings in AWS as at the moment SleakOps does not allow it. 
```

Comment:
pondria support it en vez de allow it (me imagino que seguro estan asi tambien las demas bases de datos pero just saying)